README.md

```diff
@@ -7,13 +7,13 @@ The project uses poetry for Python to create an isolated environment and manage
 ### Poetry installation (Bash)
 
 ```bash
-curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -
+curl -sSL https://install.python-poetry.org | python -
 ```
 
 ### Poetry installation (PowerShell)
 
 ```powershell
-(Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py -UseBasicParsing).Content | python -
+(Invoke-WebRequest -Uri https://install.python-poetry.org -UseBasicParsing).Content | python -
 ```
 
```

Comment:
You should update the README.md with instructions on running the app within Docker.

In particular, what are the command to run the build and run the development container with a bind mount? A development container where you have to rebuild/restart it every time you make a change is so inconvenient that I'd call it worthless.